encryption/enclib/encryption.go

```diff
@@ -146,6 +146,12 @@ func DecryptLayer(dc *config.DecryptConfig, encLayerReader io.Reader, desc ocisp
 	return commonDecryptLayer(encLayerReader, optsData)
 }
 
+// DecryptLayerKeyOptsData is used to check if we can unwrap the symmetric key
+// using given private keys required to decrypt the layer
+func DecryptLayerKeyOptsData(dc *config.DecryptConfig, desc ocispec.Descriptor) ([]byte, error) {
```

Comment:
Why do we need to add this function here? Could we just call the [keywrap interface](https://github.com/lumjjb/image/blob/encryption-policy/encryption/enclib/keywrap/keywrap.go#L27) for this?

Reply:
I added that function there because, in order to call [UnwrapKey](https://github.com/lumjjb/image/blob/bc39fc091a1a3ffec2375dc031783529f220cd83/encryption/enclib/encryption.go#L197), I will have to construct the required arguments using the logic found in [decryptLayerKeyOptsData](https://github.com/lumjjb/image/blob/bc39fc091a1a3ffec2375dc031783529f220cd83/encryption/enclib/encryption.go#L155)